modules/rest-api/src/blaze/rest_api/routes.clj

```diff
@@ -98,134 +98,134 @@
   (when-let
    [{:blaze.rest-api.resource-pattern/keys [interactions]}
     (u/resolve-pattern resource-patterns structure-definition)]
-    (cond->
```

Comment:
According to `parinfer`, the `(cond->` should be aligned with the `[` from the `when-let`.